pkg/controller/controller.go

```diff
@@ -395,16 +200,7 @@ func NewController(
 	// We follow the kubernetes way of reconciling the safety controller
 	// done by adding empty key objects. We initialize it, to trigger
 	// running of different safety loop on MCM startup.
-	controller.machineSafetyOrphanVMsQueue.Add("")
 	controller.machineSafetyOvershootingQueue.Add("")
-	controller.machineSafetyAPIServerQueue.Add("")
-
-	machineInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
-		// addMachineToSafety makes sure machine objects does not overshoot
-		AddFunc: controller.addMachineToSafety,
-		// deleteMachineToSafety makes sure that orphan VM handler is invoked
-		DeleteFunc: controller.deleteMachineToSafety,
-	})
```

Comment:
SafetyOvershooting is not moved to OOT because whenever there is a spike, the MachineDeployment is frozen by safetyovershooting controller. So, it certainly makes sense to keep it in MCM-Core.

Reply:
Oh alright, in that case its fine. So that means we'll have to keep machine informer in mcm core , so that we could add event handler which runs overshooting logic in case of machine add

Reply:
yes